### PR TITLE
added JoBaHP user

### PR DESCRIPTION
**Pull request by JoBaHP**
## Check List (Check all the applicable boxes) 

- [ x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x ] All new and existing tests passed.
- [x ] This PR does not contain plagiarized content.
- [x ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1490"></a>

### CODE REVIEW

**Review by Panquesito7 — APPROVED** [review]
Thanks for contributing! 🎉


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Maintainable

File: public/data/JoBaHP.json [hunk 1]
```diff
@@ -0,0 +1,54 @@
+{
+  "name": "Jovan R.",
+  "type": "personal",
+  "bio": "Full Stack Web Developer!",
+  "avatar": "https://github.com/JoBaHP.png",
+  "links": [
+    {
+      "name": "My business website",
+      "url": "http://build-up.us",
```
Comment (kumarsonsoff3) [inline public/data/JoBaHP.json:9]: Minor fix! Your website is using a secure protocol. 🛠

```suggestion
      "url": "https://build-up.us",
```

File: public/data/JoBaHP.json [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+      "icon": "link"
+    },
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/JoBaHP",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/https://twitter.com/joccoo",
```
Comment (kumarsonsoff3) [inline public/data/JoBaHP.json:24]: Broken link, seems like you have pasted the original link after typing the half link manually.

Small fix! 🛠

```suggestion
      "url": "https://twitter.com/joccoo",
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

**Review by kumarsonsoff3 — APPROVED** [review]
Looks Good! 👍  Thanks for contributing! 🚀

**Pull request merged**